submissions/maxim-zabolotny/pair-game/script.js

```diff
@@ -0,0 +1,150 @@
+const cardsArray = [{
+  'name': 'shell',
+  'img': 'img/blueshell.png',
+},
+{
+  'name': 'star',
+  'img': 'img/star.png',
+},
+{
+  'name': 'bobomb',
+  'img': 'img/bobomb.png',
+},
+{
+  'name': 'mario',
+  'img': 'img/mario.png',
+},
+{
+  'name': 'luigi',
+  'img': 'img/luigi.png',
+},
+{
+  'name': 'peach',
+  'img': 'img/peach.png',
+},
+{
+  'name': '1up',
+  'img': 'img/1up.png',
+},
+{
+  'name': 'mushroom',
+  'img': 'img/mushroom.png',
+},
+{
+  'name': 'thwomp',
+  'img': 'img/thwomp.png',
+},
+{
+  'name': 'bulletbill',
+  'img': 'img/bulletbill.png',
+},
+{
+  'name': 'coin',
+  'img': 'img/coin.png',
+},
+{
+  'name': 'goomba',
+  'img': 'img/goomba.png',
+},
+];
+
+const gameGrid = cardsArray
+.concat(cardsArray)
+.sort(() => 0.5 - Math.random());
+
+let firstGuess = '';
+let secondGuess = '';
+let count = 0;
+let previousTarget = null;
+let delay = 1200;
+
+const game = document.getElementById('game');
+const grid = document.createElement('section');
+grid.setAttribute('class', 'grid');
+game.appendChild(grid);
+
+gameGrid.forEach(item => {
+const { name, img } = item;
+
+const card = document.createElement('div');
+card.classList.add('card');
+card.dataset.name = name;
+
+const front = document.createElement('div');
+front.classList.add('front');
+
+const back = document.createElement('div');
+back.classList.add('back');
+back.style.backgroundImage = `url(${img})`;
+
+grid.appendChild(card);
+card.appendChild(front);
+card.appendChild(back);
+});
```

Comment:
There are several places in your code where indentation is missed for blocks of code, like here, which makes it less readable. Normally code editor does it for you automatically, e.g. in VS Code you can switch on "Editor: Format On Save" option in Settings.

Reply:
@maxim-zabolotny You might have misunderstood this. What i mean is that the code inside brackets `{...}` should have indentation to the left of it.
So for example instead of this:
```
gameGrid.forEach(item => {
const { name, img } = item;
const card = document.createElement('div');
...
});
```
it has to be like this: 
```
gameGrid.forEach(item => {
  const { name, img } = item;
  const card = document.createElement('div');
  ...
});
```